hack/llmo-dev/llm-operator-values-llmo-dev.yaml

```diff
@@ -6,3 +6,27 @@ global:
       hosts:
       - api.dev.llmo.cloudnatix.com
       secretName: api-tls
+
+  auth:
+    oidcIssuerUrl: https://api.dev.llmo.cloudnatix.com/v1/dex
+
+dex-server:
+  oauth2:
+    passwordConnector:
+      enable: false
+    responseTypes:
+    - code
+  connectors:
+  - type: oidc
+    id: okta
+    name: okta
+    config:
+      issuer: https://login.cloudnatix.com/oauth2/aus24366mbRFxF3De4x7
+      clientID: 0oak1yta82395U5wP4x7
+      clientSecret: nxU6XRatwKdweHn-CgS5YiYKNfahfS8l1N3kjhqNeWfPEAEb7ub2TfvezU5OYifH
```

Comment:
@Ladicle , @guangrui-cloudnatix , I simply put this here, but please let me know if you think there is a security risk.

Reply:
The potential risks I can think of quickly is:
Multiple dex-server can behave as the same okta client, and if anyone can log in to one of the (weak) accounts on okta, they can generate a token and can do anything.

Reply:
I see. Let me update the secret in Okta and make this one invalid.

I think dex can inject this from a secret. We can change later.